.github/workflows/test-tutorials.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+  pull_request:
```

Comment:
Most of the PRs are not effecting examples so we shouldn't run this workflow on every PR.